src/main/java/frc/robot/controls/Operator.java

```diff
@@ -85,34 +80,24 @@ public void configureControls() {
             getRightTrigger().onTrue(new Shoot(shooter, arm, drive, index));
         }
         if(shooter != null){
-            getLeftTrigger().onTrue(new PrepareShooter(shooter, Shooter.addSlip(Shooter.shooterSpeedToRPM(ShooterConstants.SHOOT_SPEED_MPS))));
+            getLeftTrigger().onTrue(new PrepareShooter(shooter, Shooter.addSlip(Shooter.shooterSpeedToRPM(ShooterConstants.SHOOT_SPEED_MPS-1))));
         }
```

Comment:
A Hack. Don't do hacks. If you do a hack, then you better add a comment that explains why.

Reply:
We forgot to change this back after making shooter speed 15 again.